src/gui/settings.cpp

```diff
@@ -1189,46 +1189,116 @@ void FurnaceGUI::drawSettings() {
       CONFIG_SECTION("Emulation") {
         // SUBSECTION LAYOUT
         CONFIG_SUBSECTION("Cores");
-        ImGui::AlignTextToFramePadding();
-        ImGui::Text("Arcade/YM2151 core");
-        ImGui::SameLine();
-        ImGui::Combo("##ArcadeCore",&settings.arcadeCore,arcadeCores,2);
+        if (ImGui::BeginTable("##Cores",3)) {
+          ImGui::TableSetupColumn("##System",ImGuiTableColumnFlags_WidthFixed);
+          ImGui::TableSetupColumn("##PlaybackCores",ImGuiTableColumnFlags_WidthStretch);
+          ImGui::TableSetupColumn("##RenderCores",ImGuiTableColumnFlags_WidthStretch);
+          ImGui::TableNextRow();
+          ImGui::TableSetColumnIndex(0);
```

Comment:
No.

Use ImGui::TableNextColumn().

Reply:
Which line?

Reply:
1197

Reply:
Should the other ImGui::TableSetColumnIndex(0)'s be changed too?

Reply:
Yeah. TableNextColumn after TableNextRow has the same effect as TableSetColumnIndex(0).